pom.xml

```diff
@@ -689,6 +689,21 @@
                     <version>1.2.1</version>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <configuration>
+                        <compilerId>javac-with-errorprone</compilerId>
```

Comment:
I am not very familiar with this compiler, and I wonder if this could have an impact on the generated bytecode, compared to the default setting. Is there a way we could run this only on test builds and not on release builds?